.github/workflows/cron-checks.yml

```diff
@@ -79,13 +77,16 @@ jobs:
       uses: actions/cache@v3
       id: runtime-cache
       with:
-        path: ~/Library/Caches/XcodeInstall/
-        key: runtime-ios-${{ matrix.ios }}
-        restore-keys: runtime-ios-${{ matrix.ios }}
+        path: ~/Downloads/*.dmg
```

Comment:
Why ~/Downloads?

Reply:
It's the default path where dmg images are saved.